Program.cs

```diff
@@ -5,6 +5,8 @@
 using NLog.Config;
 using NLog.Targets;
 using System.Globalization;
+using Newtonsoft.Json;
+using System.IO;
 
 namespace SupportBank
```

Comment:
Unlike the OutputHelper I feel like it's less clear what the responsibility of this file is. It deals with some setup, contains the main method but also has functions for reading files and adding transactions. Putting setup and the main method together makes sense but I'm not sure the rest of the functions make as much sense in this file. What do you think? Maybe something like an `InputHelper` class, to go with the `OutputHelper`, might be useful? 🙂